TaskApp/TaskApp/ViewController.swift

```diff
@@ -53,7 +53,7 @@ class ViewController: UIViewController, UITableViewDelegate, UITableViewDataSour
         
         tableView.deselectRow(at: indexPath, animated: false)
         
-        if tableView.cellForRow(at: indexPath)?.accessoryType == UITableViewCell.AccessoryType.checkmark {
+        if tableView.cellForRow(at: indexPath)?.accessoryType ==  UITableViewCell.AccessoryType.checkmark { //
```

Comment:
You can make this more readable by using local variable to store the intermediate values, so you don't have to repeat them each twice.